### PR TITLE
Post author links to author wall

### PR DESCRIPTION
**Pull request by Hives**

### CODE REVIEW

**Comment by Hives** [comment]
[Post author name links to author's wall](https://trello.com/c/mbTbTVVz/90-post-author-name-links-to-authors-wall)

**Comment by j-dav** [comment]
I approve

File: app/views/posts/index.html.erb [hunk 1]
```diff
@@ -10,7 +10,7 @@
   <div class='post-card'>
     <div class="post-header">
       <div class="post-author">
-        <%= post.author_username %>
+        <%= link_to post.author_username, "/#{post.author_username}" %>
```
Comment (JRRS1982) [inline app/views/posts/index.html.erb:13]: Loving this! Great use of accessing the author... via the post! I would not have thought of that!

File: spec/features/user_can_post_on_wall_spec.rb [hunk 2]
```diff
@@ -23,4 +23,12 @@
       expect(page).not_to have_content ("Hello user1, from user2")
     end
   end
+
+  scenario "The post's author's name links to their wall" do
```
Comment (JRRS1982) [inline spec/features/user_can_post_on_wall_spec.rb:27]: Great that you were able to TDD this, very chuffed that its 14.30 on friday, an hour before the presentation. and we are still TDDing!!!

**Review by JRRS1982 — APPROVED** [review]

**Comment by JRRS1982** [comment]
Great coding!

**Pull request merged**